test/core/excel/ExportToExcelAsString.dyn

```diff
@@ -0,0 +1,284 @@
+{
+  "Uuid": "0bcbd2b1-8e73-4563-9f92-09e8da643f87",
+  "IsCustomNode": false,
+  "Description": null,
+  "Name": "ExportToExcelAsString",
+  "ElementResolver": {
+    "ResolutionMap": {}
+  },
+  "Inputs": [],
+  "Outputs": [],
+  "Nodes": [
+    {
+      "ConcreteType": "CoreNodeModels.Input.Filename, CoreNodeModels",
+      "HintPath": "C:\\Users\\SylvesterKnudsen\\Documents\\GitHub\\StudioLEDynamo\\test\\core\\excel\\WriteFile.xlsx",
```

Comment:
I think we missed this during the PR review.. Not sure how self CI passed with this...

Reply:
@QilongTang Now that you mention it, I think @alfredo-pozo said `ExcelTests` are not run in self-serve CI. This is my bad, sorry for forgetting.

Reply:
@QilongTang @mmisol - the weird part is why did we not get a master-15 job until today that contained this code?